docs/dev/table/functions.md

```diff
@@ -2413,6 +2413,18 @@ REPEAT(string, integer)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+REGEXP_REPLACE(string1, string2, string3)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns a string resulting from replacing all substrings in <i>string1</i> that match the regular expression <i>string2</i> with replacement <i>string3</i>, if <i>string1</i> or <i>string2</i> or <i>string3</i> is NULL, returns NULL.</p> 
```

Comment:
\<p>Returns a string from \<i>string1\</i> with all the substrings that match a regular expression \<i>string2\</i> being replaced with \<i>string3\</i>.\</p>